components/icons/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@nice-digital/icons",
+  "version": "2.1.7",
+  "description": "Icons for use in NICE Digital Services",
+  "main": "index.js",
+  "scripts": {
+    "start": "npm run webfont && npm run svgr",
+    "svgr": "svgr -d es6 src && npm run es5ify",
+    "es5ify": "babel es6 -d lib",
+    "webfont": "grunt dist --color",
+    "test": "echo \"No tests :)\" && exit 0",
+    "readme": "doctoc ./README.md && babel-node ./scripts/readme-replacer.js",
+    "lint": "eslint .",
+    "// Releasing": "",
+    "release": "np",
```

Comment:
Can we remove release commands as we'll release with lerna at the root? And we don't have np dep here so would fail